zestedesavoir/ansible-zestedesavoir
# Add Google Analytics API secrets file

Add Google Analytics API secrets file

linked to https://github.com/zestedesavoir/zds-site/pull/5160

**Quality Assurance :**

- edit `group_vars/test/vars.yml` with : 
  
  ```yml
  appversion: pull/5160/head
  env: prod
  sentry_dsn: ""
  mysql:
    user: test
    password: test
    name: test
  secrets:
    google_analytics_api:
      project_id: "fake"
  ```yml
- run `vagrant up` or `vagrant provision`
- run `vagrant ssh` then `cat /opt/zds/app/api_analytics_secrets.json`
- `api_analytics_secrets.json` should contains variables from `roles/app/default/main.yml` (for instance, `type`) and from `group_vars/test/vars.yml` (here it is `project_id`)
